README.md

```diff
@@ -113,7 +113,11 @@ Left would be created with `nnoremap <silent> <c-h> :TmuxNavigateLeft<cr>`.
 
     let g:tmux_navigator_save_on_switch = 1
 
-This will execute the update command on leaving vim to a tmux pane. Default is Zero
+This will execute the `update` command on leaving vim to a tmux pane. Default is Zero
+
+    let g:tmux_navigator_save_all_buffers_on_switch = 1
```

Comment:
Yeah, that seems like a pretty reasonable idea. If you think that's better I'm more than happy to implement it.